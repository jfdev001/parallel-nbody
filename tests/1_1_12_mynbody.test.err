Reservation number 3913315: Reserved 1 hosts for 900 seconds 
Run on 1 hosts for 960 seconds from Tue Jan 17 17:46:34 2023
: node054/0 
Running N-body with 12 bodies and 1000 steps

N-body took: 0.030 seconds
Performance N-body: 0.05 GFLOPS
