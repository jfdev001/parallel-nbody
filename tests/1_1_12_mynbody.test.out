   313.023      9.014      1.471      1.763      4.578      2.841
   875.979    766.024     -6.778    -13.590     -4.299      4.837
   512.017    214.989   -288.184    703.565      3.338     -2.102
   234.008     45.986     57.474     44.854      1.621     -2.843
   842.002    746.979   -361.304  -1283.850      0.422     -4.264
   502.013    248.995  -6252.323 -24608.889      2.616     -1.142
   921.994    187.981  -3895.653   3878.172     -1.160     -3.851
   865.008    280.009 -14996.182   4685.147      1.671      1.881
    32.999    740.021   3817.685  -1986.331     -0.285      4.126
   738.011    284.001  -6867.687  20132.154      2.153      0.173
   500.005    241.976  48714.225  46729.146      1.082     -4.884
   799.977    590.014 -19922.743 -48282.142     -4.559      2.746
STATUS=PASSED
